lib/project.ts

```diff
@@ -234,9 +235,7 @@ export default class Project {
         lint: this.lint,
         audit: this.audit
       });
-      // TODO: revisit to fix promise issues
-      // tslint:disable-next-line:no-floating-promises
-      addonProject.watch({ onBuild: options.onBuild, outputDir: addonDist });
+      return addonProject.watch({ onBuild: options.onBuild, outputDir: addonDist });
```

Comment:
Feels like I'm missing something here - seems like `project.watch()` doesn't actually return anything, and the outer `Bluebird.each` here isn't actually awaited. So why the return and each here?